Resources/Prototypes/_Harmony/GameRules/roundstart.yml

```
@@ -100,3 +100,17 @@
 #    back:
 #      - MagazinePistol
 #      - MagazinePistol
+
+- type: entity
+  id: ExtendedHarmonyAnnouncement
+  parent: BaseStationEventShortDelay
```

The order is incorrect
```suggestion
  parent: BaseStationEventShortDelay
  id: ExtendedHarmonyAnnouncement
```